<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konfiguracja Global Push-To-Talk</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="inputList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold; font-size: 30px</string>
    </property>
    <property name="text">
     <string>Wybierz urządzenie</string>
    </property>
   </widget>
   <widget class="QFrame" name="controlPanel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>481</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QPushButton" name="enableBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Użyj tego urządzenia</string>
     </property>
    </widget>
    <widget class="QSlider" name="volumeSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="volumeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>461</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Głośność domyślna</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="volumeDisplay">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>68</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Dec</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggle"/>
   <addaction name="actionRefreshDevices"/>
  </widget>
  <action name="actionRefreshDevices">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Odśwież urządzenia</string>
   </property>
  </action>
  <action name="actionToggle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Włącz</string>
   </property>
   <property name="toolTip">
    <string>Włącza i wyłącza serwis</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>volumeSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>volumeDisplay</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
